src/components/common/Tag/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { HTMLAttributes } from 'react';
+
+import cn from '@utils/cn';
+
+type TagSizeType = 'small' | 'large';
+interface TagProps extends HTMLAttributes<HTMLDivElement> {
+  size: TagSizeType;
+}
+
+const tagStyleAttributes = {
+  small: 'h-[24px] w-[43px] px-[8px] caption-10-bold',
+  large: 'h-[32px] px-[12px] caption-12-bold',
+};
+
+export default function Tag({ children, className, size }: TagProps) {
+  const tagStyles = tagStyleAttributes[size];
+
+  return (
+    <div
+      className={cn(
+        `inline-flex justify-items items-center justify-between gap-[4px] rounded-[32px] py-[4px] ${tagStyles}`,
+        className,
+      )}
+    >
```

Comment:
나머지 prop을 넘겨주는 부분이 빠져있군여...!! 요것만 넣어주시면 감사하겠습니다!

```suggestion
export default function Tag({ children, className, size, ...restProps }: TagProps) {
  const tagStyles = tagStyleAttributes[size];

  return (
    <div
      className={cn(
        `inline-flex justify-items items-center justify-between gap-[4px] rounded-[32px] py-[4px] ${tagStyles}`,
        className,
      )}
      {...restProps}
    >
```

Reply:
오옥 네네 알겠습니다

Reply:
https://github.com/depromeet/ddoeat_client/pull/70/commits/75759f5821aa73dae5f383f002cc4fc039f7dcdc
요기서 반영했습니다~!